octoprint_octobullet/templates/octobullet_settings.jinja2

```diff
@@ -26,6 +26,17 @@
             <input type="text" class="input-block-level" data-bind="value: settings.settings.plugins.octobullet.push_channel">
         </div>
     </div>
+    {% trans %}<p>
+        You can define a time interval (in minutes) here for regular update messages as the print progresses. Set to 0 to prevent updates. First and last messages will usually not be sent. Be aware of the monthly message limit applied to free PushBullet accounts.
```

Comment:
Setting quiet time to 0 turns stuff off? I find this unintuitive. At least UI wise it should maybe rather be displayed as a toggle switch (which internally sets that value to 0 to signal it's disabled) which when toggled on allows setting the quiet time (> 1).

Also, I can't help myself, but I think "quiet_time" for something that is actually a "regular progress report interval" is a very unintuitive name?